docs/recipes/install/common/add_ww4_hosts_intro.tex

```diff
@@ -0,0 +1,10 @@
+%\iftoggle{isx86}{\clearpage}
+% begin_ohpc_run
+% ohpc_validation_comment Add hosts to cluster
+\begin{lstlisting}[language=bash,keywords={},upquote=true,basicstyle=\footnotesize\ttfamily,literate={BOSVER}{\baseos{}}1]
+[sms](*\#*) for ((i=0; i<$num_computes; i++)) ; do
+wwctl node add --container=rocky-9.4 \
+    --ipaddr=${c_ip[$i]} --hwaddr=${c_mac[$i]} --netmask=${internal_netmask} ${compute_prefix}$((i+1))
```

Comment:
I would suggest to use  `${c_name[i]}` instead of `${compute_prefix}$((i+1))`.

Reply:
done